File: src/main/java/com/fifo/ticketing/domain/performance/repository/GradeRepository.java
package com.fifo.ticketing.domain.performance.repository;

import com.fifo.ticketing.domain.performance.entity.Grade;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import java.util.List;

@Repository
public interface GradeRepository extends JpaRepository<Grade, Long> {
    List<Grade> findAllByPlaceId(Long id);
}

File: src/main/java/com/fifo/ticketing/domain/performance/repository/GradeRepository.kt
package com.fifo.ticketing.domain.performance.repository

import com.fifo.ticketing.domain.performance.entity.Grade
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface GradeRepository : JpaRepository<Grade, Long> {
    fun findAllByPlaceId(id: Long): List<Grade>
}
